vscode-trace-extension/src/utils/trace-extension-logger.ts

```diff
@@ -0,0 +1,186 @@
+/***************************************************************************************
+ * Copyright (c) 2023 BlackBerry Limited and others.
+ *
+ * Licensed under the MIT license. See LICENSE file in the project root for details.
+ ***************************************************************************************/
+
+import * as vscode from 'vscode';
+
+export class TraceExtensionLogger {
+    private logChannel: vscode.OutputChannel;
+    private channelDisposed = false;
```

Comment:
I think the `constructor` (below) should explicitly set `channelDisposed` to `false`, while this default being `true` until then.